src/SearchBar.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import checkSearch from "./productData";
+
+function SearchBar({ makeSearch }) {
+	function onSubmitSearch(e) {
+		e.preventDefault();
+		console.log(item);
+
+		if (item != "") {
```

Comment:
This is too complicated! Let's just check right away if it is empty and return, then submit the new filter if not.

```
if(item === "") {
    console.error(...)
    return
}

makeSearch(item)
setItem("")
```